src/App/App.js

```diff
@@ -1,12 +1,28 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { base as baseTheme, Box, Grommet, Heading, Paragraph as P } from 'grommet'
 import { deepMerge } from 'grommet/utils'
+import auth from 'panoptes-client/lib/auth'
 import zooTheme from '@zooniverse/grommet-theme'
 import Tester from '@src/App/Tester'
+import { config } from '@src/config.js'
 
 const appTheme = deepMerge(baseTheme, zooTheme)
 
 export default function App () {
+
+  async function checkUser () {
+    try {
+      const token = await auth.checkBearerToken()
```

Comment:
This will fail without a user session. Call `auth.checkCurrent()` to get any existing Panoptes session. Only call `auth.checkBearerToken()` when you need to get a token in order to do something that requires permission.